guide/14-deep-learning/object-detection.ipynb

```diff
@@ -431,7 +431,9 @@
    "source": [
```

Comment:
<p>parenthesis only for (EMD)</p><blockquote>This creates an Esri Model Definition <strong>(EMD) </strong>file that can be used for inferencing</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/549/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>